### PR TITLE
Updating `kbd`, `mark`, `code` and `pre`

### PR DESCRIPTION
**Pull request by bobdenotter**
I've changed these four HTML tags a tiny bit. A monospaced font is optically larger at the same point size, because on average the letters are much wider. This PR compensates that by making monospaced tags slightly smaller. 

After:

Screenshot 2021-01-20 at 15 39 02

Before: 

Screenshot 2021-01-20 at 15 39 38

I've noticed the indentation of the CSS is all over the place if you're not using an IDE that 'hides' that from you: '2 spaces', '4 spaces' and '1 tab' are used together. After this one is merged in, I'd like to make a PR to fix the formatting, if that's OK with you. It'll be a large delta, without functional changes. 

Screenshot 2021-01-20 at 15 34 26

### CODE REVIEW

**Comment by kevquirk** [comment]
Hmmm it must be hiding it from me somewhat - I'm using Atom. Here's how it looks for me:



I see the additional tab for `pre code` but not the rest. Weird. I'm happy for you to go through and fix this is you wish.

**Comment by bobdenotter** [comment]
Editors like Atom are indeed 'helping' by showing it more consistently.. You can see it on github, if you edit the file: https://github.com/kevquirk/simple.css/edit/main/simple.css (and scroll down to the bottom).. 

I'll happily clean it up after this one's merged in (otherwise it'll get merge conflicts)

**Review by kevquirk — CHANGES_REQUESTED** [review]
Agree that `kbd` shouldn't use the same pink, but I don't like the `--preformatted` variable. The `#666` doesn't look right and it doesn't work in dark mode.

I think a better solution is to get rid of `--preformatted` and simply replace it with `--text` so that the text is the same colour and uniform.

Happy with the other changes.

**Comment by kevquirk** [comment]
Also changed `--marked` to `#FFF59D`

**Comment by bobdenotter** [comment]
> but I don't like the --preformatted variable. The #666 doesn't look right and it doesn't work in dark mode.

Do you not like the addition of the variable itself? Or just the color I've picked? I overlooked dark mode, so i'll fix that for sure! 

> Also changed --marked to #FFF59D

**Comment by kevquirk** [comment]
Hey Bob,

Sorry for the delay in reply.

> Do you not like the addition of the variable itself? Or just the color I've picked? I overlooked dark mode, so i'll fix that for sure! I might want to have another go at it, and if you still don't like it, i'll just as happily remove it!

I think `#666` just doesn't work with the rest of theme. We don't have that kind of mid-grey anywhere else. Making `kbd` the same colour as the `body` text and having the "key" effect on the border, as well as the mono font is enough separate it from the normal text, yet have it fit in nicely with the theme.

What do you think?

> On your screenshot the code actually does look to be too small now. :-/

Yeah, I had it at `1rem` on the original version IIRC, and felt the same - that it looked too small, so ended up making that `1.15rem` too. Maybe `1.05rem` or something to make it look closer to the body text?

Thanks for all your contributions, they really are helping to improve this project!

**Comment by bobdenotter** [comment]
I've update the CSS: 

 - I did keep the `--preformatted`, but made it work correctly on both light  and dark. Also, it differs _less_ from `--text`. Let me know it you like it, otherwise i'll remove it. Your call. 
 - `mark` works better now. Also with Dark Mode
 - Font size on code/pre updated. Looks more consistent on Windows vs. MacOS now. 

Current screenshots: 

Schermafbeelding 2021-01-22 om 17 14 25
Schermafbeelding 2021-01-22 om 17 14 34

**Comment by kevquirk** [comment]
The contrast for `mark` in dark mode still isn't right it's only 3.8:1. I think if we keep it as `#FFDD33` for light and dark mode, then set `color` to always be `#212121` for `mark` that should do it.

**Comment by kevquirk** [comment]
Like this:

**Comment by bobdenotter** [comment]
Sounds good. Done! :-)

**Review by kevquirk — APPROVED** [review]

**Pull request merged**
